netbox/extras/views.py

```diff
@@ -885,3 +886,24 @@ class JobResultBulkDeleteView(generic.BulkDeleteView):
     queryset = JobResult.objects.all()
     filterset = filtersets.JobResultFilterSet
     table = tables.JobResultTable
+
+
+#
+# Markdown
+#
+
+class MarkdownRenderView(View):
+    # _ignore_model_permissions = True
```

Comment:
Rather than going through the extra steps of defining and rendering a template, we can call the `render_markdown` template filter directly on the input value.

Additionally, if the value is empty, we should return an empty string (rather than the placeholder) to ensure the view is suitable for general use.